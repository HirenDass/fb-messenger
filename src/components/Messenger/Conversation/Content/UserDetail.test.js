import React from 'react'
import { shallow, mount } from 'enzyme'
import UserDetail from './UserDetail'
import { shallowToJson } from 'enzyme-to-json'

describe('<UserDetail />', () => {
  it('renders properly',() => {
    const wrapper = shallow(<UserDetail secondsAgo={1} toggleModal={() =>{}} />)

    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
