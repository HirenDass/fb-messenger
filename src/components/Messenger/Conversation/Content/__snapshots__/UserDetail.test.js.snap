// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserDetail /> renders properly 1`] = `
<styled.div>
  <styled.div>
    <div>
      <Avatar
        size="large"
        username="alex"
      />
      <div>
        <styled.div>
          alex
        </styled.div>
        <styled.div>
          Active 
          1
          m ago
        </styled.div>
      </div>
    </div>
    <a
      onClick={[Function]}
    >
      <Icon
        name="cog"
      />
    </a>
  </styled.div>
  <div>
    Options
  </div>
  <div>
    Facebook Profile
  </div>
</styled.div>
`;
